### PR TITLE
Introduce observed_estimated_label

### PR DESCRIPTION
**Pull request by arkid15r**
## Proposed change

While reviewing https://github.com/vacanza/python-holidays/pull/1615 I noticed cases with the extra brackets "(estimated) (observed)" label text inconsistency. This PR is aimed to address the first problem.

## Type of change

- [ ] New country/market holidays support (thank you!)
- [ ] Supported country/market holidays update (calendar discrepancy fix, localization)
- [x] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency update (version deprecation/upgrade)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] I've run `make pre-commit`, it didn't generate any changes
- [x] I've run `make test`, all tests passed locally

[contributing-guidelines]: https://github.com/vacanza/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/vacanza/python-holidays/tree/beta/docs/source

### CODE REVIEW

File: holidays/locale/en_US/LC_MESSAGES/BN.po [hunk 1]
```diff
@@ -78,3 +80,8 @@ msgstr "Islamic New Year"
 #. Birth of the Prophet
 msgid "Maulidur Rasul"
 msgstr "Birth of the Prophet"
+
+#. %s (observed, estimated).
+#, c-format
+msgid "%s (diperhatikan, anggaran"
```
Comment (KJhellico) [inline holidays/locale/en_US/LC_MESSAGES/BN.po:86]:
```suggestion
msgid "%s (diperhatikan, anggaran)"
```

File: holidays/locale/ms/LC_MESSAGES/BN.po [hunk 2]
```diff
@@ -78,3 +80,8 @@ msgstr ""
 #. Birth of the Prophet
 msgid "Maulidur Rasul"
 msgstr ""
+
+#. %s (observed, estimated).
+#, c-format
+msgid "%s (diperhatikan, anggaran"
```
Comment (KJhellico) [inline holidays/locale/ms/LC_MESSAGES/BN.po:86]:
```suggestion
msgid "%s (diperhatikan, anggaran)"
```

File: holidays/locale/th/LC_MESSAGES/BN.po [hunk 3]
```diff
@@ -78,3 +80,8 @@ msgstr "วันขึ้นปีใหม่อิสลาม"
 #. Birth of the Prophet
 msgid "Maulidur Rasul"
 msgstr "วันเมาลิดนบี"
+
+#. %s (observed, estimated).
+#, c-format
+msgid "%s (diperhatikan, anggaran"
+msgstr "%ชดเชย%s, โดยประมาณ"
```
Comment (KJhellico) [inline holidays/locale/th/LC_MESSAGES/BN.po:87]:
```suggestion
msgid "%s (diperhatikan, anggaran)"
msgstr "ชดเชย%s, โดยประมาณ"
```
Maybe `msgstr "ชดเชย%s (โดยประมาณ)"`, for consistency with other Thai estimated names translations (with brackets).
@PPsyrius, we need your expertise here.

**Review by KJhellico — COMMENTED** [review]
i18n support is something awesome... 🤪

File: holidays/locale/th/LC_MESSAGES/BN.po (hunk 3 above)
Comment (PPsyrius) [reply holidays/locale/th/LC_MESSAGES/BN.po:87]: `ชดเชย%s (โดยประมาณ)` would be the correct translation here

Comment (arkid15r) [reply holidays/locale/th/LC_MESSAGES/BN.po]: 👍

File: holidays/locale/ms/LC_MESSAGES/BN.po (hunk 2 above)
Comment (arkid15r) [reply holidays/locale/ms/LC_MESSAGES/BN.po]: Great eye!

**Review by arkid15r — COMMENTED** [review]
I've addressed the comments and added an extra check for estimated/observed_estimated labels

**Review by KJhellico — APPROVED** [review]
LGTM!

**Pull request merged**